src/common/syncjournaldb.h

```diff
@@ -68,6 +68,7 @@ class OCSYNC_EXPORT SyncJournalDb : public QObject
 
     void keyValueStoreSet(const QString &key, qint64 value);
     qint64 keyValueStoreGetInt(const QString &key, qint64 defaultValue);
```

Comment:
maybe time to use QVariant or other way to also store other data types ?

Reply:
@mgallien I can do it for the future if needed, but, right now I don't see a point. All I need is to get a number of failed checksum validations for a specific checksum used as a key and a `qint64` as a value. Why do I need a QVariant in this case? Isn't it overkill when we just need to store an int?